mesh/node.c

```diff
@@ -1566,6 +1565,48 @@ static void get_managed_objects_cb(struct l_dbus_message *msg, void *user_data)
 
 		cb(node, agent);
 
+	} else if (req->type == REQUEST_TYPE_IMPORT) {
+
+		node_ready_func_t cb = req->cb;
+		struct node_import_request *import_data = req->user_data;
```

Comment:
Extract `mesh_db_import` to a local variable, call to `add_local_node` will be shorter.